pytket/extensions/qiskit/backends/aer.py

```diff
@@ -182,11 +199,13 @@ def _arch_independent_default_compilation_pass(
             return SequencePass([DecomposeBoxes(), SynthesiseTket()])
         return SequencePass([DecomposeBoxes(), FullPeepholeOptimise()])
 
-    def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
+    def default_compilation_pass(
+        self, optimisation_level: int = 2, placement_options: Optional[Dict] = None
+    ) -> BasePass:
```

Comment:
Please add a docstring here. Can just describe `placement_options` as "optional dictionary containing arguments to `NoiseAwarePlacement`" or similar. With the change suggested above, if those arguments every change or others are added, this will still work.

Reply:
Ignore the above, I've simply linked to the IBMQBackend docsumentation to avoid dupliaction here.